Use veclib on ppc64le and diable the conflicting types from altivec.h
--- vendor/spoa/src/simd_alignment_engine.cpp.orig	2020-02-11 10:18:20.037978000 +0000
+++ vendor/spoa/src/simd_alignment_engine.cpp	2020-02-11 10:18:45.457371805 +0000
@@ -9,7 +9,9 @@
 #include <limits>
 
 extern "C" {
-    #include <immintrin.h> // AVX2 and lower
+    #include "vec128intlib.h"
+    #undef bool
+    #undef vector
 }
 
 #include "spoa/graph.hpp"
